rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+
+The following rules apply to a `SetRequest` containing `origin` CLI.  TODO:
+Special considerations for mixing native schema are currently undefined.
```

Comment:
Please don't leave a TODO like this. It's pretty unclear what this sentence means.

What does "mixing native schema" mean? If there's something specific that we are not discussing then that's fine (mixing a native YANG-modelled schema with a non-native one? overlapping contents between two YANG-modelled schemas?), but this sentence doesn't make a whole lot of sense.

Reply:
I added in response to several comments requesting we expand the scope of this PR to also clarify mixing vendor yang models.  I will remove.